readme added

10shubham01 opened the pull request.

Commit:
readme added

Commit:
implemented post api in addCard

Commit:
fixed v-app

Commit:
deletion done

Commit:
implemented UUID

Commit:
vuex implemented

jasmine-sandhu17 reviewed (commented):
` <card v-for="post in posts" :key="post.id" :initialData="post" />`
why <card> is being used, why not <v-card>?

jasmine-sandhu17 reviewed (changes requested):
` <card
        v-for="post in posts"
        :key="post.id"
        :initialData="post"
        v-on:deleteCard="deleteCard($event)"
      />`
      use v-card

jasmine-sandhu17 reviewed (commented):
` v-on:deleteCard="deleteCard($event)"`
or 
` @deleteCard="deleteCard($event)"`

jasmine-sandhu17 reviewed (changes requested):
`async filterCard(id) {
      console.log(id);
      const api = `https://jsonplaceholder.typicode.com/posts?userId=${id}`;
      const filteredCard = await axios.get(api).then((resp) => resp.data);
      this.posts = filteredCard;
    },`
    Remove console
    `search: function (e) {
      if (e.keyCode == 13) {
        console.log(e.target.value);
        this.filterCard(e.target.value);
      }
    },`
    use one syntax for defining functions. In the same file, u are using two syntaxes.

Commit:
filnal commit

All review points implemented ..
search bar bug fixed,
merged add card and updatecard,
snackbar added to addcard

Commit:
snackbar position changed to top

10shubham01 commented:
> ` <card v-for="post in posts" :key="post.id" :initialData="post" />` why is being used, why not ?
resolved

10shubham01 commented:
> ` <card v-for="post in posts" :key="post.id" :initialData="post" v-on:deleteCard="deleteCard($event)" />` use v-card

resolved

10shubham01 commented:
> ` v-on:deleteCard="deleteCard($event)"` or ` @deleteCard="deleteCard($event)"`

resolved

10shubham01 commented:
> `async filterCard(id) { console.log(id); const api = `[https://jsonplaceholder.typicode.com/posts?userId=${id}`](https://jsonplaceholder.typicode.com/posts?userId=$%7Bid%7D%60); const filteredCard = await axios.get(api).then((resp) => resp.data); this.posts = filteredCard; },`Remove console`search: function (e) { if (e.keyCode == 13) { console.log(e.target.value); this.filterCard(e.target.value); } },` use one syntax for defining functions. In the same file, u are using two syntaxes.

resolved

Commit:
functions and file name fixed
